pages/docs/overview/faq.md

```diff
@@ -227,6 +229,9 @@ Additionally, there are precautions within the container context to mitigate any
 
 You can read more about the Singularity <a href="/docs-security">security overview here</a>.
 
+### When I try mounting my container with the ```-B``` or ```--bind``` option I receive an <i>unknown option</i> or <i>Invalid argument</i> error.
+
+Make sure that you are using the most recent Singularity release to mount your container to the host system.  Also, make sure you are using an up-to-date Singularity to bootstrap your container.  Some features (such as ```--bind```) will not work in earlier versions.
 
```

Comment:
I really like this addition - I (still) run into this from time to time, but because I put the `--bind` in the wrong spot! Should we add a bit about that as well? Something like:

Make sure that you are using the most recent Singularity release to mount your container to the host system, and that the `--bind` argument is placed after the execution command (e.g, run, exec, shell). An example might look like this:

      singularity run -B $PWD:/data

What do you think, something like that? Then I think we are good to go!